src/core/SampleBuffer.cpp

```diff
@@ -935,39 +903,41 @@ f_cnt_t SampleBuffer::getPingPongIndex( f_cnt_t _index, f_cnt_t _startf, f_cnt_t
 }
 
 
-void SampleBuffer::visualize( QPainter & _p, const QRect & _dr,
-							const QRect & _clip, f_cnt_t _from_frame, f_cnt_t _to_frame )
+void SampleBuffer::visualize (QPainter & p, const QRect & dr,
+							  const QRect & clip, f_cnt_t from_frame, f_cnt_t to_frame)
```

Comment:
Curious on thoughts about long parameter lists that go past line length:
1. just leave it a long line (ignore line length)
```suggestion
void SampleBuffer::visualize(QPainter & p, const QRect & dr, const QRect & clip, f_cnt_t from_frame, f_cnt_t to_frame)
```
2. line per parameter
```suggestion
void SampleBuffer::visualize(
	QPainter & p,
	const QRect & dr,
	const QRect & clip,
	f_cnt_t from_frame,
	f_cnt_t to_frame
)
// end parenthesis could be combine with the last parameter, alternatively combine the opening brace with the close parenthesis, e.g.
// ) {
```
3. the oddball, but I've seen some libraries that do this, separate the return type, function name, and then parameters
```suggestion
void
SampleBuffer::visualize(
 /* parameter list same as 2 */
) {
```

Reply:
I prefer 2! About the code style review, I think it would be easier for @russiankumar (and myself when dealing with the conflicts :laughing: ) if we leave less critical style fixes for #5791 . `SampleBuffer.cpp` has tons of them and I had to go through the whole file so I tried to fix most. The parameter list of `visualize` is one I think I fixed on the cpp file but not on the header. Will update it later!

Reply:
The main thing here is removing the space between the function name and the opening parenthesis. If we touch the line it needs to be updated to the new convention, period. I was a huge stickler about it before, and then everyone complained that it didn't matter and code should look the same as the surrounding, and I backed down since everyone was so adamant about it, but I'm tired of seeing people ignore it and I'm going to request fixes to conform to the new style. 😛

As for the ideas there, just curious what people thought. 😄

Reply:
I didn't mean to imply you were nitpicking, hope it didn't sound like that. I was just working in parallel in a code style revamping on that file that could fix that too. But you're totally right, it should be fixed on the PR that touches the line and we should be strict about it!

Reply:
> As for the ideas there, just curious what people thought. 😄

Leaving this open in case anyone wants to drop their thoughts.